am-lib/src/test/java/uk/gov/hmcts/reform/amlib/internal/utils/SecurityClassificationTest.java

```diff
@@ -13,17 +15,29 @@
 import static uk.gov.hmcts.reform.amlib.enums.SecurityClassification.PUBLIC;
 import static uk.gov.hmcts.reform.amlib.enums.SecurityClassification.RESTRICTED;
 
-@SuppressWarnings({"PMD.UnusedPrivateMethod", "LineLength"})
+@SuppressWarnings({"PMD.UnusedPrivateMethod", "LineLength", "PMD.ArrayIsStoredDirectly"})
 class SecurityClassificationTest {
 
     @ParameterizedTest
-    @MethodSource("createArguments")
-    void whenRoleSecurityClassificationIsHighEnoughResourceShouldBeVisible(boolean expectedResult, SecurityClassification roleClassification, SecurityClassification resourceClassification) {
+    @MethodSource("visibleArguments")
+    void whenRoleSecurityClassificationIsHighEnoughResourceShouldBeVisible(boolean expectedResult,
+                                                                           SecurityClassification roleClassification,
+                                                                           SecurityClassification resourceClassification) {
         boolean isVisible = resourceClassification.isVisible(roleClassification.getHierarchy());
         assertThat(isVisible).isEqualTo(expectedResult);
     }
 
-    private static Stream<Arguments> createArguments() {
+    @ParameterizedTest
+    @MethodSource("fromValueArguments")
+    void fromValueOfShouldReturnSetOfSecurityClassifications(SecurityClassification securityClassification,
```

Comment:
Test could use rename since main method got renamed. Same with static method playing role of data source.